project.clj

```diff
@@ -1,6 +1,6 @@
 (def ks-version "1.0.0")
 
-(defproject puppetlabs/trapperkeeper "1.2.1-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper "1.3.0-SNAPSHOT"
```

Comment:
Nah, because of the HUP support... doesn't that seem like a new feature that should qualify as a Y?